Special abilities v2, mostly Flak (#4)
GameBrains/er-core: modules/special-rules/pages/save.adoc

= Save

Due to exceptionally resilient armour, protective fields, or extreme skills and toughness, these troops are capable of surviving almost any impact.

---

Apply these rules to units with the {save} ability:

* Each time the unit suffers a hit, from any source, roll a die. On a roll of 4+, the hit is ignored and causes no damage.

> * The {save} ability applies to attacks that ignore void shields and power fields.

Author: Just spotted that the Q&A for Save allows it to be used against destruction via your Transport vehicle dying, even though that is **not** a hit.

This is just rough text to act as a placeholder/reminder.

```suggestion
* The {save} ability applies to attacks that ignore void shields and power fields.
* The {save} ability applies to transported units when their {transport} vehicle is destroyed.
```

Author: Hmm. Looking at this again, I think it would be cleaner for the Transport rules to say that embarked units all suffer a hit when the Transport is destroyed. I'll post on FB to see if anyone can spot rules issues with that approach.

If destroyed Transports result in a 'hit' on the embarked units, this added bullet point can simply be left out. _Maybe_ adding a tip or note as a reminder instead.

Author: The Transport page has been updated to say that the units suffer hits, as per the FB discussions. So this is suggestion is no longer needed.

= Save

These units can beat the odds and escape destruction -- perhaps through exceptional armour, protective fields, or extreme skills and toughness.

---

Where your unit has the {save} ability:

* Each time the unit suffers a hit, from any source, roll a die.
On a roll of 4+, ignore the hit -- it causes no damage.

> NOTE: Where void shields and power fields are in effect, apply the {save} ability only to attacks that ignore these.

Author: This bit is now completely bass-ackwards. 🤣

There are no units with a Save and with void shields or power fields. This is a reminder that attacks that bypass void shields or power fields, for example Distortion Cannon or close combat hits, don't bypass Saves.

```suggestion
NOTE: Apply the {save} ability against all hits, even to attacks that ignore void shields or power fields.
```

Reviewer: Okay, it's great when you catch this stuff as 'rules SME'. The original text seemed ambiguous/unclear to me, and it seems that while my new sentence may be clearer it is also wrong. :-D  This kind of thing will happen sometimes when I rework the text as 'technical writer', so catching these is an important part of your 'rules SME' role.

However, to be clear — does the Save ability apply to hits on void shields and power fields? I didn't check, but I rather thought that it does not? You suggestion implies that Save should apply to shields/fields also ("all hits").

Author: Revised version which should be less ambiguous:

```suggestion
NOTE: Attacks which ignore void shields or power fields do *not* ignore the {save} ability.
```
